File: QuantConnect.AlphaStream/Requests/GetAlphaOrdersRequest.cs
using QuantConnect.AlphaStream.Infrastructure;
using QuantConnect.AlphaStream.Models.Orders;
using RestSharp;
using System.Collections.Generic;

namespace QuantConnect.AlphaStream.Requests
{
    /// <summary>
    /// Fetch the Alpha Orders and OrderEvents list(backtest and live trading) track record since publication.
    /// </summary>
    [Endpoint(Method.GET, "alpha/{id}/orders")]
    public class GetAlphaOrdersRequest : AttributeRequest<List<AlphaStreamOrder>>
    {
        /// <summary>
        /// Unique id hash of an Alpha published to the marketplace.
        /// </summary>
        [PathParameter("id")]
        public string Id { get; set; }

        /// <summary>
        /// Starting position for the search query. With very large datasets the results can be paginated and fetched in chunks starting from start.
        /// </summary>
        [QueryParameter("start")]
        public int Start { get; set; }
    }
}

File: AlphaStream/Requests/GetAlphaOrdersRequest.py
class GetAlphaOrdersRequest:
    """ Fetch an alpha's orders in batches, starting from `start` """

    def __init__(self, alphaId, start=0):
        self.Id = alphaId
        self.Start = start
        self.Endpoint = "alpha/{}/orders".format(alphaId)

    def GetPayload(self):
        payload = {
            "id": self.Id,
            "start": self.Start
        }
        return payload